src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+$secondary-color-gray: #616070;
+$buttun-color: #00acdc;
+$margin-center: 0 auto;
+$family-roboto: "Roboto", "Helvetica", sans-serif;
+
 body {
+  font-family: $family-roboto;
+  color: #060b35;
   margin: 0;
 }
+
+.card {
+  margin: $margin-center;
+  height: 406px;
+  width: 200px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)

when all elements have the same `box-sizing` value, their behavior becomes more predictable